release-notes/opensearch-security.release-notes-2.17.0.0.md

```diff
@@ -0,0 +1,41 @@
+## Version 2.17.0 Release Notes
+
+Compatible with OpenSearch and OpenSearch Dashboards version 2.17.0
+
+### Enhancements
+* Add `ignore_hosts` config option for auth failure listener ([#4538](https://github.com/opensearch-project/security/pull/4538))
+* added API roles for correlationAlerts ([#4689](https://github.com/opensearch-project/security/pull/4689))
+* Manually backport #4632 (Allow multiple signing keys to be provided) ([#4666](https://github.com/opensearch-project/security/pull/4666))
```

Comment:
```suggestion
* Allow multiple signing keys to be provided ([#4666](https://github.com/opensearch-project/security/pull/4666))
```

Reply:
Done in: [4ebe0ed](https://github.com/opensearch-project/security/pull/4705/commits/4ebe0ed29bb91ae02dedb8779d87fbedd53b782f)